source/_patterns/01-atoms/list-group/demo/list-groups.twig

```diff
@@ -0,0 +1,209 @@
+{# This function loops through the scssColors global variable to make an array of all the available colors. #}
+{% set colors = [] %}
+{% for name, rgb in scssColors %}
+  {% set item_classes = {
+    item_color: name
+  } %}
+  {% set colors = colors|merge([name]) %}
+{% endfor %}
+
+
+<div class="container">
+  {% embed '@atoms/grid/_grid-1-up.twig' %}
+    {% block column_1 %}
+      <div class="mb-3">
+        <h1 class="text-center">List Group</h1>
+        {% include '@molecules/card/_card-helper.twig' with {
+          card_text: list_group_helper,
+        } %}
+      </div>
+
+      {# Basic examples #}
+      <div class="mb-3">
+        {% include '@molecules/card/_card-helper.twig' with {
+          card_title: 'Basic Examples',
+          card_text: list_group_helper_basic,
+        } %}
+      </div>
+      <div class="row mb-3">
+        <div class="col-sm-6">
+          {% include '@atoms/list-group/_list-group.twig' with list_group_text %}
+        </div>
+        <div class="col-sm-6">
+          {# This block of code is going to appear multiple times. It exists because the _list-group.twig
+          template ultimately needs an object to iterate through to generate the individual items. These code
+          blocks exist to pull data from multiple disperate places (the all_colors array, list-groups.yml,
+          etc.) and combine them into a single object that the template can then process. #}
+          {% set color_items = [] %}
+          {# Iterate over colors, since this is actually what we're demo'ing #}
+          {% for color in colors %}
+            {# Inline object merged into random text object #}
+            {% set color_item = { item_color: color } | merge(random(list_group_text.items)) %}
```

Comment:
If everything else is going to use `temp_item` instead of their specific case (ie `active_item`, `disabled_item`, `action_item`) then ensure all are consistent. So change this `color_item` (or the others) to be a consistent naming pattern.